### ISSUE TITLE
Mice need way too much food/drink.

### ISSUE THREAD

**Issue by asperger-sind** [space-wizards/space-station-14#10594]
## Description

You are a mouse, a small, tiny, cute miserable little shitter designed to rest tightly in the pocket of their owner, or walk around the station consuming food.
You, just so happen, to be very small, and weight not a lot, except for the fact that you can eat through the entire kitchen, botany, every department with their donk pockets and pizzas, and still not feel full, not even close, no, you won't even feel that you ate all that, you'll at some point start starving even if you ate food throughout that entire timespan whilst doing it with supreme efficiency.

**Additional context**

tl;dr, mice need to eat much, much less to actually sustain themselves, unlike how it's currently unintentionally(?) realized wherein you need to eat 400 pounds of food to actually go out of the starving debuff for 2 minutes. (not like you can do the 400 pounds, you can eat 2-3 bites per cycle AT MOST)

**Comment by OctoRocket** [comment]
This could be done by changing the baseDacayRate but I would need
a) for this to be confirmed as something that needs to be done (I think it is)
b) for someone to tell me how hunger decay works

**Comment by OctoRocket** [comment]
That PR didn't change anything. The merged status doesn't mean anything here because nothing was changed.
